aspnetcore/blazor/fundamentals/configuration.md

```diff
@@ -19,7 +19,29 @@ Blazor WebAssembly loads configuration from:
 * App settings files by default:
   * `wwwroot/appsettings.json`
   * `wwwroot/appsettings.{ENVIRONMENT}.json`
-* Other [configuration providers](xref:fundamentals/configuration/index) registered by the app. Not all providers are appropriate for Blazor WebAssembly apps. Clarification on which providers are supported for Blazor WebAssembly is tracked by [Clarify configuration providers for Blazor WASM (dotnet/AspNetCore.Docs #18134)](https://github.com/dotnet/AspNetCore.Docs/issues/18134).
+* Other [configuration providers](xref:fundamentals/configuration/index) registered by the app. Not all providers or provider features are appropriate for Blazor WebAssembly apps:
```

Comment:
After looking at the preview of the docs page in my browser, I feel like this information would be better suited under its own heading instead of in a bullet.

Reply:
I think one whole level of bullets can be collapsed here. I'll try it and we can see how it looks.